mq/main/comm-util/src/main/java/com/sun/messaging/jmq/util/MultiColumnPrinter.java

```diff
@@ -326,16 +327,14 @@ public void addTitle(String[] row) {
      * </PRE>
      *
      * @param row Array of strings to print in one row of title.
-     * @param span Array of integers that reflect the number of collumns the corresponding title string will occupy.
+     * @param span Array of integers that reflect the number of columns the corresponding title string will occupy.
      */
-    public void addTitle(String[] row, int span[]) {
+    public void addTitle(String[] row, Integer span[]) {
         // Need to create a new instance of it, otherwise the new values will
         // always overwrite the old values.
 
         String[] rowInstance = new String[(row.length)];
-        for (int i = 0; i < row.length; i++) {
-            rowInstance[i] = row[i];
-        }
+        System.arraycopy(row, 0, rowInstance, 0, row.length);
```

Comment:
Why not `j.u.Arrays.copyOf`?

Reply:
This was what the IDE suggested. It is also is what Arrays.copyOf does eventually, just with some extra steps,